protostar/__init__.py

```diff
@@ -3,7 +3,9 @@
 from typing import Any
 
 from crypto_cpp_py.cpp_bindings import cpp_hash
-from starkware.starknet.services.api.contract_class import ContractClass
+from starkware.starknet.services.api.contract_class.contract_class import (
+    DeprecatedCompiledClass,
```

Comment:
Previous implementation was renamed to `Deprecated...`.
Many of implementations relating to cairo0 did rename in that manner.